Perlang.Common/Perlang.Common.csproj

```diff
@@ -19,4 +19,8 @@
       <NoWarn>1591;1701;1702</NoWarn>
     </PropertyGroup>
 
+    <ItemGroup>
+      <PackageReference Include="JetBrains.Annotations" Version="2020.2.0-eap" />
```

Comment:
(These make it possible to get rid of "unused class" and "unused method" warnings in IDEs like JetBrains Rider.)